ArcSlider/ArcSlider.js

```diff
@@ -206,14 +214,17 @@ const ArcSliderBase = kind({
 		style: ({radius, style}) => {
 			const size = ri.scaleToRem(radius * 2);
 			return {...style, height: size, width: size};
-		}
+		},
+		circleRadius: ({isFocused}) => isFocused ? 20 : 15,
+		backgroundColor: ({backgroundColor, skinVariants}) => backgroundColor || (skinVariants.night ? '#444444' : '#888888'),
+		foregroundColor: ({foregroundColor, skinVariants}) => foregroundColor || (skinVariants.night ? '#ffffff' : '#000000')
```

Comment:
Please check before accessing the property.

```suggestion
		backgroundColor: ({backgroundColor, skinVariants}) => backgroundColor || (skinVariants && skinVariants.night ? '#444444' : '#888888'),
		foregroundColor: ({foregroundColor, skinVariants}) => foregroundColor || (skinVariants && skinVariants.night ? '#ffffff' : '#000000')
```